server/src/main/resources/db/changelog/20210412145432-drop_content_cache.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20210412145432-1" author="ojanus">
+        <sql>UPDATE cp_cert_serial SET expiration = NOW() WHERE id IN (SELECT serial_id FROM cp_cont_access_cert)</sql>
```

Comment:
Could the results returned from that subquery hit a limit with either mariadb/postgres for amount of args in the IN clause?
Perhaps another way of writing this query would be safer

Reply:
Done.